config/manager/service.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: kfserving-controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: controller-manager-service
```

Comment:
We should drop the prefixing of ko for these.

Reply:
Which prefixes are you referring to ?

Reply:
All these resources get kfserving appended to them

kfserving + controller-manager-service.

I wish these yamls we're just named what they should be named. I don't see the value of prefixing them.